### PR TITLE
Every user bound to cf_user role in the root ns

### PR DESCRIPTION
**Pull request by Birdrock**
## Is there a related GitHub Issue?
#721 

## What is this change about?
When roles are created (rolebindings are created), users get an additional `cf_user` rolebinding

## Does this PR introduce a breaking change?
No.

## Acceptance Steps
Run all tests as normal.

## Tag your pair, your PM, and/or team
@clintyoshimura @tcdowney @gcapizzi

### CODE REVIEW

**Comment by Birdrock** [comment]
@gcapizzi I'm curious about a scenario where we are creating a ServiceAccount role. We implemented this for the case where an SA would also need the `cf_user` role binding, but are unsure whether this is proper.

**Comment by gcapizzi** [comment]
@Birdrock yes, I think this behaviour should be in place for service accounts as well.

File: api/apis/integration/role_test.go [hunk 1]
```diff
@@ -21,11 +21,13 @@ var _ = Describe("Role", func() {
 	)
 
 	BeforeEach(func() {
+		// TODO: This creates an arbitrary mapping that is not representative of production mapping
```
Comment (tcdowney) [inline api/apis/integration/role_test.go:24]: I assume the mappings are tested for real by the e2e tests when we do things like this:
https://github.com/cloudfoundry/cf-k8s-controllers/blob/a1dd31293d3c0cd59bd3cd792f058ac693c95cd4/api/tests/e2e/apps_test.go#L43-L44

Regarding the "TODO" are we actually planning on changing this here or should we just make this a "NOTE" or something.

Comment (Birdrock) [reply api/apis/integration/role_test.go:24]: The real mappings are exercised in production code, which the e2e tests hit.

Regarding the "TODO" - I'm not sure what to put here, but wanted to highlight that the map keys needed to exist, but their values are irrelevant in the integration tests.

**Review by clintyoshimura — APPROVED** [review]
LGTM

**Pull request merged**